src/proto/grpc/testing/stats.proto

```diff
@@ -41,6 +41,12 @@ message ServerStats {
   // change in server time (in seconds) used by the server process and all
```

Comment:
We've had it this way since the beginning but I can't think of the reason why. We are actually only able to measure it in micros since that's what getrusage provides.